packages/document/doc-tools-doc/docs/en/plugin/introduction.mdx

```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+The plugin system is a crucial part of Modern.js Doc, which allows you to easily extend the abilities of the framework during the process of building a site. So, what ability can you extend with plugins? Let's take a look at the overall architecture of Modern.js first.
+
+The overall architecture of Modern.js Doc is shown in the figure below:
+
+![Modern.js Doc Architecture](/architecture-en.png)
```

Comment:
It is recommended to put large pictures on CDN. If we put them in the repo, the size of repo will be large in the future, resulting in slower clone.